docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -115,6 +115,7 @@ After activating the option to use the configuration file:
 
 -   Codacy will use the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results will take those changes into account.
 -   If Codacy analyzes a branch that doesn't include the configuration file, Codacy reverts to using the code patterns configured for the tool before you selected the option **Configuration file** on the Code patterns page.
+-   Codacy still uses the configuration file even if you ignore it using either [the Codacy UI](ignoring-files.md) or the [Codacy configuration file](codacy-configuration-file.md).
```

Comment:
Minor suggestion, altough not blocking
```suggestion
-   Codacy will use tool configuration files even if you ignore them through [Codacy's UI](ignoring-files.md) or in a [Codacy configuration file](codacy-configuration-file.md).
```

Reply:
Thank you @MiguelMarcelino! Here's one more edit building on top of your suggestion to avoid the apostrophe on "Codacy's" and also improve the link text. :+1: 

```suggestion
-   Codacy will use tool configuration files even if you ignore them [using the Codacy UI](ignoring-files.md) or [in a Codacy configuration file](codacy-configuration-file.md).
```